### PR TITLE
Added relentless task.

### PR DESCRIPTION
**Pull request by siad007**
Inspired by [ant-contrib](http://doc.zengrong.net/apache/ant-contrib/manual/tasks/relentless.html)

### CODE REVIEW

File: docs/docbook5/en/source/appendixes/coretasks.xml [hunk 1]
```diff
@@ -3421,6 +3421,88 @@ verbose="true" failonerror="false" /></programlisting>
 &lt;echo>Resolved [absolute] path: ${absolute_path}&lt;/echo></programlisting>
         </sect2>
     </sect1>
+    <sect1 role="taskdef" xml:id="Relentless">
+        <title>Relentless</title>
+        <para> RetrThe &lt;relentless> task will execute all of the nested tasks, regardless of whether one or more of
```
Comment (kenguest) [inline docs/docbook5/en/source/appendixes/coretasks.xml]: Typo? "RetrThe"

**Comment by siad007** [comment]
@kenguest would you mind continue the review?

**Comment by mrook** [comment]
@siad007 can you address @kenguest 's comment and resolve the conflicts?

**Comment by siad007** [comment]
@mrook missing docs added and resolved conflicts

**Pull request merged**